### DISCUSSION TITLE
Auto start with komorebi

### DISCUSSION THREAD

**Discussion by Only-Moon** [amnweb/yasb#57] [General]
last time I used yasb by running it through Python (when the launcher was not a thing) and I used the below code in the startup folder to run Yasb and Komorebi on startup. Now I'm wondering if can I run the current version of yasb through the terminal, if not then what are the alternatives 

```yaml
taskkill /f /fi "WINDOWTITLE eq YasbBar"
cmd /c start /min "" komorebic stop
taskkill /f /im "whkd.exe"
cmd /c start /min "" komorebic start --whkd
Start-Process -WindowStyle Hidden pythonw "C:\Users\mohit\Downloads\.dotfiles\src\main.pyw"
```

**Comment by amnweb** [comment]
I will add YASB in env. variable as path so you can run it from console, like `yasb`, now you can just type in powershell `C:\Users\YOUR_USERNAME\AppData\Local\Yasb\yasb.exe` to run YASB.

**Comment by amnweb** [comment]
YASB CLI is available now, usage `yasbc help`
